packages/teams-js/src/public/authentication.ts

```diff
@@ -371,6 +371,9 @@ export namespace authentication {
   export function notifySuccess(result?: string, callbackUrl?: string): void {
     redirectIfWin32Outlook(callbackUrl, 'result', result);
     ensureInitialized(FrameContexts.authentication);
+    if (GlobalVars.isFramelessWindow) {
```

Comment:
This is a breaking change for Mobile. We use this API AAD Auth flow in Mobile clients